docs/user_guide/HOWTO/Central_Log_Collection/Logstash.rst

```diff
@@ -367,6 +367,10 @@ to your Kibana node.
    On your puppet master, you can install the toml gem by executing
    ``puppetserver gem install toml``.
 
+   Alternatively, if internet access is not available, the gem can be installed
```

Comment:
This should now be done by default when the module RPM is installed.

We're going to have to go split brain `r10k` vs `rpm` all over the place. That said, you're right that this needs to be updated to reflect the new changes.

Reply:
Oh I forgot it did that. Updating.